src/Imgix/UrlBuilder.php

```diff
@@ -65,65 +74,96 @@ public function createURL($path, $params=array()) {
         return $uh->getURL();
     }
 
-    public function createSrcSet($path, $params=array()) {
+    public function createSrcSet($path, $params=array(), $start=100, $stop=8192, $tol=8, $disable_variable_quality=false) {
```

Comment:
Should we have `disable_variable_quality` written in camelCase? [[1]](https://blog.sideci.com/5-php-coding-standards-you-will-love-and-how-to-use-them-adf6a4855696)

Reply:
Good catch! Yes! It should be `disableVariableQuality`, thank you! I will make this change right now.

Reply:
Okay, I have updated and camel-cased the flag: `disableVariableQuality`